packages/webview_flutter/webview_flutter_android/lib/src/android_webview_controller.dart

```diff
@@ -849,6 +860,8 @@ class AndroidNavigationDelegate extends PlatformNavigationDelegate {
   PageEventCallback? _onPageStarted;
   ProgressCallback? _onProgress;
   WebResourceErrorCallback? _onWebResourceError;
+  void Function({bool didCrash, int rendererPriorityAtExit})?
```

Comment:
Also, if you want to include `rendererPriorityAtExit` in this PR, we should create an enum for this class that is converted from the constant values rather than passing an `int`. You can also just choose to add `didCrash` for this PR and can add `rendererPriorityAtExit` in a later PR.

Reply:
Changed `onRenderProcessGone` to return `RenderProcessGoneInfo` class. Also removed `rendererPriorityAtExit`, as that can be added later.